webpack/assets/javascripts/react_app/components/FormAutocomplete/form-autocomplete.scss

```diff
@@ -0,0 +1,7 @@
+.form-autocomplete {
+  .autocomplete-aux {
+    position: absolute;
+    right: 26px;
+    top: 3px;
```

Comment:
why is this needed? (just wondering why it worked before without it?)

Reply:
It was for fixing the clear button position, I will try to fix it in the `SearchBar` scss